README.txt

```diff
@@ -87,8 +87,7 @@ lib [This folder contains 3rd party libraries needed in the project]
 		Example: cd bin

```

Comment:
in reference to 3.2 and 2.2.2 of README file, There wouldn’t be any bin folder initially and this would be created when we execute the makefile. I guess terminal should be at AlternateBitProtcol folder. 
Also, I would suggest to expand the path in the example. i.e., cd documents/AlternateBitProtocol/bin. It provides more details and easier understanding to the users.

Reply:
Thank you so much for pointing that mistake. It has been solved

Reply:
contents are perfect, I guess you need to interchange the number from 3.2 to 3.3

Reply:
Yeah and do some modification